### PR TITLE
Remove `endliquid` as suggested auto-completion

### PR DESCRIPTION
**Pull request by aswamy**
## What are you adding in this PR?

Fixes https://github.com/Shopify/theme-tools/issues/479

## Before you deploy

- [x] I ran `yarn build` and committed the updated configuration files
- [x] I included a patch bump `changeset`

### CODE REVIEW

File: packages/theme-language-server-common/src/completions/providers/LiquidTagsCompletionProvider.spec.ts [hunk 1]
```diff
@@ -44,6 +44,15 @@ describe('Module: LiquidTagsCompletionProvider', async () => {
     await expect(provider).to.complete('{% form "cart", cart %} ... {% end', ['endform']);
   });
 
+  it('should not complete literal `liquid` tag', async () => {
+    await expect(provider).to.complete(
+      `{% liquid
+        echo 'hello'
+        e`,
+      ['echo'],
```
Comment (graygilmore) [inline packages/theme-language-server-common/src/completions/providers/LiquidTagsCompletionProvider.spec.ts:52]: Would it make sense to do something like `expect(provider).not.to.complete('code', ['endliquid'])`? Or maybe that would end up with a false positive. Almost wish there was a slightly different assertion that was like:

```ts
const completionOptions = provider.complete('code');
expect(completionOptions).not.toContain('endliquid')`
```

File: yarn.lock [hunk 2]
```diff
@@ -4259,11 +4259,6 @@ ms@2.0.0:
   resolved "https://registry.yarnpkg.com/ms/-/ms-2.0.0.tgz#5608aeadfc00be6c2901df5f9861788de0d597c8"
   integrity sha512-Tpp60P6IUJDTuOq/5Z8cdskzJujfwqfOTkrwIwj7IRISpnkJnT6SyJ4PCPnGMoFjC9ddhal5KVIYtAt97ix05A==
 
-ms@2.1.2:
```
Comment (graygilmore) [inline yarn.lock:4262]: I've noticed this while working locally, too. Probably safe to commit but thought I'd flag it to double check.

**Review by graygilmore — APPROVED** [review]

**Review by karreiro — APPROVED** [review]
Thank you, @aswamy! :)

File: packages/theme-language-server-common/src/completions/providers/LiquidTagsCompletionProvider.spec.ts (hunk 1 above)
Comment (charlespwd) [reply packages/theme-language-server-common/src/completions/providers/LiquidTagsCompletionProvider.spec.ts:52]: Funny you say this, in my Liquid Tag snippet completion PR, I modified the assertion to work with top level async matchers. I think that would make it possible with the not to complete with an expect.arrayContaining().

https://github.com/Shopify/theme-tools/pull/507/files#diff-77716827f5743abd97b0ab800c5b78475604beb65b8b5f21a56a5f18d8b82765

But, I mean... this test looks fine to me. The list of tags here is hardcoded at the top of the file, so this test would only break by manual intervention.

Comment (graygilmore) [reply packages/theme-language-server-common/src/completions/providers/LiquidTagsCompletionProvider.spec.ts:52]: > The list of tags here is hardcoded at the top of the file, so this test would only break by manual intervention.
nice! didn't realize that

**Pull request merged**